specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json

```diff
@@ -1935,7 +1935,11 @@
         "staticIP": {
           "type": "string",
           "pattern": "^\\d+\\.\\d+\\.\\d+\\.\\d+$",
-          "description": "Static IP address. Required when deploying a Redis cache inside an existing Azure Virtual Network."
+          "description": "Static IP address. Required when deploying a Redis cache inside an existing Azure Virtual Network.",
```

Comment:
Its not required. Its Optional.

Reply:
We need to add "readOnly" or "x-ms-mutability" or "default" to fix s360 issue.
Else we have to suppress the error.
"x-ms-mutability" seems more suitable here, since we can set It on creation and we can read.

Reply:
modified